webdriver_manager/utils.py

```diff
@@ -84,6 +84,10 @@ def get_filename_from_response(response, name):
 
 
 def console(text: str, level=logging.INFO):
+    os_wdm_log_level = os.getenv('WDM_LOG_LEVEL')
```

Comment:
Any default value here?

Reply:
Not yet. Does it need?

Reply:
As I understand there has to be INFO level as default

Reply:
Actually you have to pass it inside a method as level

Reply:
> As I understand there has to be INFO level as default

Yes, INFO is default.

> Actually you have to pass it inside a method as level

You mean this
```python 
def console(text: str, level=logging.INFO):
```
change to this?
```python
def console(text: str):
    level=logging.INFO
    os_wdm_log_level = os.getenv('WDM_LOG_LEVEL')
    if os_wdm_log_level:
        level = int(os_wdm_log_level)
```

Reply:
```
def console(text: str):
    os_wdm_log_level = int(os.getenv('WDM_LOG_LEVEL', logging.INFO))
```

Reply:
Thank you. Now i see. I agree, it looks much better :)

Reply:
To be honest, i thought about separate `console()` log levels inside wdm, that's why i used `level` as `console()` argument. But ok, we can always return to this idea later.

Reply:
Done at https://github.com/SergeyPirogov/webdriver_manager/pull/106/commits/ff6695e64b165d5d0bf47b368655f5458a5fc85b